From cde409004ec894d7651a73ba1986ad2efe3683b5 Mon Sep 17 00:00:00 2001
From: dschinazi <dschinazi@google.com>
Date: Mon, 20 Sep 2021 13:41:32 -0700
Subject: [PATCH] Add missing include

This change was proposed by Stephan Hartmann at <https://quiche-review.googlesource.com/c/quiche/+/10840>. I've confirmed that they've signed the CLA.

PiperOrigin-RevId: 397831819
---
 http2/adapter/window_manager.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/http2/adapter/window_manager.h b/http2/adapter/window_manager.h
index f15982d4..5a7701e9 100644
--- a/net/third_party/quiche/src/http2/adapter/window_manager.h
+++ b/net/third_party/quiche/src/http2/adapter/window_manager.h
@@ -1,6 +1,8 @@
 #ifndef QUICHE_HTTP2_ADAPTER_WINDOW_MANAGER_H_
 #define QUICHE_HTTP2_ADAPTER_WINDOW_MANAGER_H_
 
+#include <stddef.h>
+
 #include <functional>
 
 #include "common/platform/api/quiche_export.h"
-- 
2.31.1

